.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: build
+
+on:
+  release:
```

Comment:
What does this mean?

Reply:
Actually it didn't work...
Changed it to on:push:tags: "*"